### PR TITLE
iexec-worker PR 597, 598 and 601 repatriation

### PR DESCRIPTION
**Pull request by Natchica**
Add also pubsub package parts for succesfull build

### CODE REVIEW

**Review by thewhitewizard — APPROVED** [review]

File: src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java [hunk 1]
```diff
@@ -419,5 +430,12 @@ void shouldAllTeeSessionGenerationErrorHaveMatch() {
                     .isNotNull();
         }
     }
+
+    @Test
+    void shouldFailedAndReturnNull() {
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java]: Typo on method name here ?
```suggestion
    void shouldFailAndReturnNull() {
```

File: src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java [hunk 2]
```diff
@@ -419,5 +430,12 @@ void shouldAllTeeSessionGenerationErrorHaveMatch() {
...
+        TeeSessionGenerationError error = mock(TeeSessionGenerationError.class);
+        when(error.name()).thenReturn("NA");
+        Assertions.assertThat(preComputeService.teeSessionGenerationErrorToReplicateStatusCause(error)).isNull();
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/compute/pre/PreComputeServiceTests.java]: For consistency with other updates in same class ?
```suggestion
        assertThat(preComputeService.teeSessionGenerationErrorToReplicateStatusCause(error)).isNull();
```

File: src/test/java/com/iexec/standalone/result/ResultServiceTests.java [hunk 3]
```diff
@@ -72,39 +68,45 @@
...
-                                                        + File.separator + "test"
-                                                        + File.separator + "resources"
-                                                        + File.separator + "tmp"
-                                                        + File.separator + "test-worker"
-                                                        + File.separator;
+            + File.separator + "test"
+            + File.separator + "resources"
+            + File.separator + "tmp"
+            + File.separator + "test-worker"
+            + File.separator;
```
Comment (jbern0rd) [inline src/test/java/com/iexec/standalone/result/ResultServiceTests.java:88]: Is this due to Windows compatibility considerations ?

File: src/main/java/com/iexec/standalone/pubsub/SubscriptionService.java [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.standalone.pubsub;
+
+import com.iexec.standalone.notification.TaskNotification;
+import com.iexec.standalone.pubsub.StompClientService;
```
Comment (jbern0rd) [inline src/main/java/com/iexec/standalone/pubsub/SubscriptionService.java]: Should not be necessary as the class is in the same package

File: src/test/java/com/iexec/standalone/result/ResultServiceTests.java (hunk 3 above)
Comment (Natchica) [reply src/test/java/com/iexec/standalone/result/ResultServiceTests.java:88]: Yes it is

**Review by jbern0rd — APPROVED** [review]
Thanks

**Pull request merged**